src/android/NetworkManager.java

```diff
@@ -240,6 +241,9 @@ private String getType(NetworkInfo info) {
             if (type.equals(WIFI)) {
                 return TYPE_WIFI;
             }
+            else if (type.toLowerCase().equals(ETHERNET)) {
```

Comment:
Why add ETHERNET? Why not just:
`if (type.toLowerCase().equals(TYPE_ETHERNET)) { return TYPE_ETHERNET } `